runners/direct-java/src/main/java/org/apache/beam/runners/direct/EmptyInputProvider.java

```diff
@@ -42,7 +42,8 @@
    * as appropriate.
    */
   @Override
-  public Collection<CommittedBundle<?>> getInitialInputs(AppliedPTransform<?, ?, ?> transform) {
+  public Collection<CommittedBundle<?>> getInitialInputs(
+      AppliedPTransform<?, ?, ?> transform, int ignored) {
```

Comment:
best to keep the original name of targetParallelism even though the variable is unused